test/library/standard/IO/readConcurrent.compopts

```diff
@@ -0,0 +1 @@
+-sReadBinaryArrayReturnInt=true -suseNewFileReaderRegionBounds=true
```

Comment:
The fileReaderRegionBounds argument will generate a deprecation message on main, you'll want to merge with main and re-run this test before merging